requirements.txt

```diff
@@ -7,5 +7,6 @@ PyYAML==6.0
 markupsafe==2.0.1
 google-cloud-logging==3.1.2
 google-api-python-client==2.55.0
-goblet-gcp-client==0.1.2
+goblet-gcp-client==0.1.3
 protobuf==3.20.3
+google-cloud-tasks==2.10.4
```

Comment:
dont think we need this requirement, since we are using the goblet gcp client instead